cdap-docs/reference-manual/source/http-restful-api/lifecycle.rst

```diff
@@ -831,10 +836,12 @@ trigger again until the schedule is resumed.
 To *resume* a schedule means that the trigger is reset, and the program associated will
 run again at the next scheduled time.
 
+As a schedule is initially deployed in a *suspended* state, a call to this API is needed to *resume* it.
+
 To suspend or resume a schedule use::
 
-  POST <base-url>/namespaces/<namespace>/apps/<app-id>/schedules/<schedule-name>/suspend
-  POST <base-url>/namespaces/<namespace>/apps/<app-id>/schedules/<schedule-name>/resume
+  POST <base-url>/namespaces/<namespace>/apps/<app-id>/schedules/<schedule-id>/suspend
```

Comment:
Lets keep it name. From the way we create schedule here: http://docs.cask.co/cdap/3.2.0-SNAPSHOT/en/developers-manual/building-blocks/schedules.html?highlight=schedules name will be more appropriate. 
It was my bad to call them id in the jira.